homeassistant/components/camera/__init__.py

```diff
@@ -361,13 +360,12 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
 
     async def preload_stream(_event: Event) -> None:
         for camera in component.entities:
-            camera_prefs = prefs.get(camera.entity_id)
+            camera_prefs = await prefs.get_dynamic_stream_settings(camera.entity_id)
```

Comment:
Is there a better name for this? dynamic_stream_settings? camera_stream_prefs? stream_prefs?